pages/k8s/install-manual.md

```diff
@@ -280,35 +294,34 @@ You can also [download the fragment here][asset-aws-overlay].
 [Juju documentation][juju-bundle]. In this example it merely adds a new application,
 specifying the charm to use, and further specifies the relationships to add.
 
-To use this overlay with the **CDK** bundle, it is specified during deploy like this:
+To use this overlay with the **Charmed Kubernetes** bundle, it is specified
+during deploy like this:
 
 ```bash
-juju deploy charmed-kubernetes  --overlay ~/path/aws-overlay.yaml
-```
-
-Substitute in the local path and filename to point to your YAML fragment.
-
-Note that you will still need to run the command to share credentials with this charm:
-
-```bash
-juju trust aws-integrator
+juju deploy charmed-kubernetes  --overlay ~/path/aws-overlay.yaml --trust
 ```
 
+Substitute in the local path and filename to point to your YAML fragment. Note
+that this overlay requires credential access, so you must use the `--trust`
```

Comment:
It isn't the overlay that requires the credentials. This is pedantic, but I think it is important to be clear for people that don't understand all the concepts yet. "Note that the overlay includes a charm that requires credential access..."

Reply:
fair enough, I will reword